### DISCUSSION TITLE
Do I need to install mysql server before following the docker way ?

### DISCUSSION THREAD

**Discussion by zhousy29** [gogs/gogs#7357] [Q&A]
I  just went through the installation part , and below is confusing me :

> Make sure you install the [prerequisites](https://gogs.io/docs/installation) first.

>There are 6 ways to install Gogs:

>[Install from binary](https://gogs.io/docs/installation/install_from_binary.html)
>[Install from source](https://gogs.io/docs/installation/install_from_source.html)
>[Install from packages](https://gogs.io/docs/installation/install_from_packages.html)
>[Ship with Docker](https://github.com/gogs/gogs/tree/main/docker)
>[Try with Vagrant](https://github.com/geerlingguy/ansible-vagrant-examples/tree/master/gogs)

Do I need to install mysql server before following the docker way ?  
I take it for granted that  mysql server is pre-installed in the docker image/container, and that prerequisite of mysql is unnecessary.

hope somebody can clarify this , thanks.

**Comment by annpocoyo** [comment]
Can't you just use SQLite.
